doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -46,108 +81,268 @@ The features and capabilities described in the software and hardware profiles ar
-
-There are a number of different container runtimes. The simplest form, low-level container runtimes, just manage the OS capabilities such as cgroups and namespaces, and then run commands from within those cgroups and namesapces. An example of this type of runtime is runc, which underpins many of the higher-level runtimes and is considered a reference implementation of the [Open Container Initiative (OCI) runtime spec](https://github.com/opencontainers/runtime-spec). This specification includes details on how an implementation (i.e. an actual container runtime such as runc) must, for example, configure resource shares and limits (e.g. CPU, Memory, IOPS) for the containers that Kubernetes (via the kubelet) schedules on that host. This is important to ensure that the features and capabilities described in [chapter 5 of the RM](../../../ref_model/chapters/chapter05.md) are supported by this RA and delivered by any downstream Reference Implementations (RIs) to the instance types defined in the RM.
-
-Where low-level runtimes are used for the execution of a container within an OS, the more complex/complete high-level container runtimes are used for the general management of container images - moving them to where they need to be executed, unpacking them, and then passing them to the low-level runtime, which then executes the container. These high-level runtimes also include a comprehensive API that other applications (e.g. Kubernetes) can use to interact and manage the containers. An example of this type of runtime is containerd, which provides the features described above, before passing off the unpacked container image to runc for execution.
-
-For Kubernetes the important interface to consider for container management is the [Kubernetes Container Runtime Interface (CRI)](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/). This is an interface specification for any container runtime so that it is able to integrate with the kubelet on a Kubernetes Node. The CRI decouples the kubelet from the runtime that is running in the Host OS, meaning that the code required to integrate kubelet with a container runtime is not part of the kubelet itself (i.e. if a new container runtime is needed and it uses CRI, it will work with kubelet). Examples of this type of runtime include containerd (with cri plugin) and cri-o, which is built specifically to work with Kubernetes.
-
-To fullfill `req.inf.vir.01` the architecture should support a container runtime which provides the isolation of Operating System kernels.
-
-The architecture must support a way to isolate the compute resources of the infrastructure itself from the workloads compute resources.
+The Container Runtime is the component that runs within a Container Host
```

Comment:
```suggestion
The Container Runtime is the component that runs within a Kubernetes Node Operating System (OS) and manages the underlying OS functionality, such as cgroups and namespaces (in Linux), in order to provide a service within which container images can be executed and make use of the infrastructure resources (compute, storage, networking and other I/O devices) abstracted by the Kubernetes Node OS, based on API instructions from the kubelet.
```
I think we should consistently use Kubernetes Node.

Reply:
Fixed in https://github.com/cntt-n/CNTT/pull/1894/commits/710f0d7e5289c9a7559c2d6dabb4ecc80e46c76d (single line)